analytic_schedule/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
```

Comment:
```suggestion
# Copyright 2017-23 ForgeFlow S.L.
```
Same in all files

Reply:
@AaronHForgeFlow  I think, It looks okay to me. When I change the end date for any of the child project, code is finding max of end date of Childs and update it to parent project.
Or Maybe I understood your test case incorrectly?

Reply:
Hi @DhavalTforgeFlow, I made a mistake in the answer. What I meant to say is:

The project start PP1 date is 2023/01/01
The project end date PP1 is 2023/02/01
The SSP1 start date is changed to be 2023/01/15 --> this changed the start date in PP1 but that is not correct because the start date was already before.